src/app/not-found.tsx

```diff
@@ -0,0 +1,21 @@
+import LinkButton from '@/components/LinkButton'
+import Image from 'next/image'
+
+export default function NotFound() {
+    return (
+        <div>
+            <Image
+                src="/404.png"
+                alt='404'
+                width={800}
+                height={800}

```

Comment:
画像のheightも修正の余地あり

(元画像が1000x500で横幅800pxあれば十分そう)
```suggestion
                width={800}
                height={400}
```

Reply:
対応しました
https://github.com/megane-s/minshumi-frontend/pull/216/commits/45e01427bee68b82a67d76063493b068d983886f